File: YoonJeong/20주차/카펫.py
def solution(brown, yellow):
    answer = []
    
    mulB = brown + yellow # brown의 가로 * 세로
    sumB = (brown + 4) // 2 # brown의 가로 + 세로

    for i in range(sumB) :
        if i * (sumB - i) == mulB :
            answer.append(i)
            break
    answer.insert(0, sumB - answer[0])

    return answer

File: juhee2/20주차/카펫.java
class Solution {
    public int[] solution(int brown, int yellow) {
        int[] answer = {0,0};
        int area = brown + yellow;
        for (int i=1; i< area/ 2; i++) {
            if (area % i ==0) {
                int x = area / i;
                int y = i;
            
                if (2*(x+y)== brown +4) {
                    answer[0] = x;
                    answer[1] = y;
                    break;
                }
            }           
        }
        return answer;
    }
}
